Description: Updates keystore and db location
Author: Abmar Barros <abmar@buddycloud.com>
Last-Update: 2014-06-13

--- a/mediaserver.properties.example	2013-02-14 16:10:06.000000000 +0100
+++ b/mediaserver.properties.example	2013-02-15 03:25:55.535622561 +0100
@@ -6,7 +6,7 @@
 http.tests.port=9090
 http.endpoint=https://api.buddycloud.org/media-proxy
 https.port=8443
 https.enabled=false
-https.keystore.path=/$HOME/.jetty/jetty.jks
+https.keystore.path=/usr/share/buddycloud-media-server/jetty.jks
 https.keystore.type=JKS
 https.keystore.password=password
@@ -28,15 +28,15 @@
 xmpp.reply.timeout=30000

 # JDBC
-jdbc.db.url=jdbc:postgresql://localhost:5432/buddycloud-media-server?user=postgres&password=postgres
+jdbc.db.url=jdbc:postgresql://localhost:5432/#DATABASE_NAME#?user=#DATABASE_USER#&password=#DATABASE_PASSWORD#
 jdbc.driver.class=org.postgresql.Driver

 # File System
-media.storage.root=/tmp
+media.storage.root=/usr/share/buddycloud-media-server/storage

 # Max threshold beyond which files are written directly to disk, in bytes
 # Only used while uploading multipart form data files
 media.todisk.threshold=1048576

 # Max filesize in bytes
 media.sizelimit=100000000
