Description: Create database tokens for dbconfig
Author: Abmar Barros <abmar@buddycloud.com>
Last-Update: 2013-08-06

--- a/configuration.properties.example	2013-08-06 16:00:06.000000000 +0100
+++ b/configuration.properties.example	2013-08-06 16:01:53.862662029 +0100
@@ -9,6 +9,6 @@
-jdbc.proxool.driver-url=jdbc:hsqldb:.
+jdbc.proxool.driver-url=jdbc:postgresql://localhost:5432/#DATABASE_NAME#
-jdbc.proxool.driver-class=org.hsqldb.jdbcDriver
+jdbc.proxool.driver-class=org.postgresql.Driver
-jdbc.user=username
+jdbc.user=#DATABASE_USER
-jdbc.password=password
+jdbc.password=#DATABASE_PASSWORD#
 jdbc.proxool.maximum-connection-count=10
 jdbc.proxool.house-keeping-test-sql=select CURRENT_DATE
 