Description: Fix path to default release dir
Author: Abmar Barros <abmar@buddycloud.com>
Last-Update: 2013-01-30

--- a/contrib/buddycloud-http-init
+++ b/contrib/buddycloud-http-init
@@ -24,7 +24,7 @@
 # It has been tested under CentOS 5, but should work OK in other linux
 # distros.
 
-PREFIX=/opt
+PREFIX=/usr/share
 PROG_NAME=buddycloud-http-api
 PROG_DIR=$PREFIX/$PROG_NAME
 PID_PATH=/var/run/${PROG_NAME}.pid
