Description: Fix path to default config
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2012-10-23

--- a/src/main.coffee
+++ b/src/main.coffee
@@ -10,7 +10,7 @@
 # Config
 config = require('jsconfig')
 version = require('./version')
-defaultConfigFile = path.join(__dirname,"..","config.js")
+defaultConfigFile = path.join(__dirname,"config.js")
 if fs.existsSync(defaultConfigFile)
     config.defaults defaultConfigFile
 
