Description: Fix path to running directory and default config
Author: Abmar Barros <abmar@buddycloud.com>
Last-Update: 2013-01-30

--- a/_etc_init.d_buddycloud-server
+++ b/_etc_init.d_buddycloud-server
@@ -22,9 +22,9 @@
 DESC="buddycloud server"
 NAME=buddycloud
 RUN_AS_USER=buddycloud
-RUNDIR=/opt/buddycloud-server
+RUNDIR=/usr/share/buddycloud-server
 DAEMON=`which nodejs`
-DAEMON_ARGS="lib/main.js --config config.js"
+DAEMON_ARGS="main.js --config /etc/buddycloud-server/config.js"
 PIDDIR=/var/run/buddycloud-server
 PIDFILE="$PIDDIR/$NAME.pid"
 SCRIPTNAME=/etc/init.d/$NAME
